Pocket Casts Watch App Extension/FilterEpisodeListView.swift

```diff
@@ -61,11 +61,10 @@ struct FilterEpisodeListView: View {
                     Spacer()
                 }
             } else {
-                ScrollView {
-                    LazyVStack(alignment: .leading) {
-                        header
+                List {
+                    Section(content: {
```

Comment:
[nit] We can omit the content here. 

```suggestion
                    Section {
```

Reply:
Remove unnecessary content parameter name